index.html

```diff
@@ -0,0 +1,304 @@
+                  ><img src="icons/angelist.png" alt="angelist icon"
+                /></a>
+              </li>
+            </ul>
+
+            <button type="button" class="btn resume">Get my resume</button>
+          </div>
+          <ul class="language-section-container">
+            <li class="language-section open">
+              <div class="language-title-section">
+                <button type="button" class="drop-down down">Language</button>
+                <img src="./icons/expanded.svg" alt="expanded arrow" />
+              </div>
+
+              <ul class="language-section-box-sec">
+                <li class="language-section-box">
+                  <img src="./images/pic1.png" alt="javaScript logo" />
+                  <p class="language-name">JavaScript</p>
+                </li>
+                <li class="language-section-box">
+                  <img src="./images/pic2.png" alt="html logo" />
+                  <p class="language-name">HTML</p>
+                </li>
+                <li class="language-section-box">
+                  <img src="./images/pic3.png" alt="css logo" />
+                  <p class="language-name">Css</p>
+                </li>
+              </ul>
+            </li>
+            <li class="language-section">
+              <div class="language-title-section">
+                <button type="button" class="drop-down down">Framework</button>
+                <img src="./icons/not-expanded.svg" alt="not-expanded arrow" />
+              </div>
+            </li>
+            <li class="language-section">
+              <div class="language-title-section">
+                <button type="button" class="drop-down down">Skills</button>
+                <img src="./icons/not-expanded.svg" alt="not-expanded arrow" />
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <!-- contact-section -->
+      <section class="contact-form">
+        <div class="contact-form-section">
+          <div class="contact-form-card">
+            <h2 class="contact-form-title">Contact me</h2>
+            <p class="contact-form-description">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <form id="form" action="ttps://formspree.io/f/mjvlgvab" method="post">
+            <label for="username"
+              ><input
+                type="text"
+                required
+                placeholder="Full Name"
+                maxlength="30"
+                id="username"
+            /></label>
+            <label for="Email"
+              ><input
+                type="Email"
+                id="Email"
+                name="Email"
+                required
+                maxlength="30"
+                placeholder="Your Email"
```

Comment:
[OPTIONAL] Please make the inputs for `name` and `email` take the same width as the ` Textarea ` to match the Figma design